### DISCUSSION TITLE
Building a Content Warning Module with Drew Brasher - Orchard Core Pair Programming by Lombiq

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#16047] [Show and tell]
We'll have the third session of Orchard Core Pair Programming by Lombiq! In these, we do an hour of pair programming with an Orchard Core community member about a project of theirs. We learn together a lot, share best practices, and write some good code. All this is live, and you can join us with your questions! Check it out here: https://www.youtube.com/watch?v=ECEbkLSOqLQ. See you at 5 PM UTC on the 21st! (On a Tuesday due to Whit Monday in Europe.)

This time, [Drew Brasher](https://github.com/DrewBrasher) will join us with a Content Warning Module project that she started in this discussion: https://github.com/OrchardCMS/OrchardCore/discussions/13643. We'll start from at least a scaffolding of the module, take the https://www.aaron-gustafson.com/notebook/considering-content-warnings-in-html/ blog post as a guideline, and then build it from there.

Your host and the "navigator" of the pair programming session will be me.

Would you like to be our guest? Just write a comment here!

**Comment by DrewBrasher** [comment]
Here is the scaffolded module and test web project that I will start with: https://github.com/DrewBrasher/OrchardCoreModules

**Comment by Piedone** [comment]
We're starting in five minutes! Tune in here: https://www.youtube.com/watch?v=ECEbkLSOqLQ
